lib/zendesk_apps_support/package.rb

```diff
@@ -131,11 +282,9 @@ def readified_js(app_name, app_id, asset_url_prefix, settings = {}, locale = 'en
           asset_url_prefix: asset_url_prefix,
           app_class_name: app_class_name,
           author: author,
-          translations: app_translations(locale),
+          translations: translations[locale],
```

Comment:
is there much of a difference between app_translations and translations? the app_translations method still exists.